bakerydemo/base/models.py

```diff
@@ -132,7 +100,7 @@ class Meta:
 class StandardPage(Page):
     """
     A generic content page. On this demo site we use it for an about page but
-    could be used for any type of page content.
+    it could be used for any type of page content.
```

Comment:
How about "could be used for any basic page that doesn't need custom data fields." ? ("any type of page content implies that we could use StandardPage to create pages with custom content fields).